### ISSUE TITLE
tslint not parsing semicolons correctly when readonly properties used.

### ISSUE THREAD

**Issue by dsifford** [palantir/tslint#1275]
### Bug Report (delete this for feature requests)

- __TSLint version__: (not sure -- using in Atom) -- linter-tslint version 0.9.3. I assume this is the latest version?
- __TypeScript version__: 1.9.0 (nightly)
- __Running TSLint via__: Atom

#### TypeScript code being linted

```ts
interface TEST {
    readonly x: number;
};
```

with `tslint.json`:

Using this rule: `"semicolon": [true, "always"]` (among others)

#### Actual behavior

Warning: semicolon - missing semicolon

#### Expected behavior

No warning

**Comment by JKillian** [comment]
@dsifford I added a test [here](https://github.com/palantir/tslint/pull/1276) to show that this doesn't happen in `tslint@next`, which is the version of TSLint that goes with the nightlies. Most likely `linter-tslint` is pulling in a version of TypeScript that is 1.8.x or older, and that's where the issue you're seeing is arising from. If you run TSLint on your code manually w/ TS 1.9.x, I don't think you'll see the error

**Comment by dsifford** [comment]
@JKillian Yep, that was it! Thanks 👍
